plugindocs.rb

```diff
@@ -27,7 +35,13 @@ def execute
 
     alias_definitions = Util::AliasDefinitionsLoader.get_alias_definitions
 
-    repositories.peach(parallelism) do |repository_name, details|
+    # we need to sort to be sure we generate docs first for embedded plugins of integration plugins
+    # and skip the process for stand-alone plugin if already processed
+    sorted_repositories = repositories.sort_by { |name,_|  name.include?('-integration-') ? 0 : 1 }
+    sorted_repositories.peach(parallelism) do |repository_name, details|
```

Comment:
```suggestion
    if parallelism > 1
      $stderr.puts("WARN: --parallelism is temporarily constrained to 1\n")
    end
    sorted_repositories.each do |repository_name, details|
```

Reply:
>Instead of holding this useful fix up in a forever-queue, I believe we can temporarily constrain the parallelism to 1 for this execution mode (read: not VPR), and go with your as-proposed changes for ensuring that we handle integration plugins before stand-alones.

^^ Yes, please. 😄 
I will need to make manual fixes to every PR for every branch until this gets merged.  Recent examples:
* https://github.com/elastic/logstash-docs/pull/1426
* https://github.com/elastic/logstash-docs/pull/1427

Reply:
Agreed! Placed some comments and applied the change.